### PR TITLE
Fixed partition naming for anbox

### PR DESCRIPTION
**Pull request by NeoTheThird**
Why is there no standards for this? I wish for a vigilante that goes around punishing android vendors for doing stupid bullshit.

### CODE REVIEW

**Comment by NeoTheThird** [comment]
huh, should not have pushed that to the same branch, because i squashed #55 there are empty commits in here. Just squash this one as well and we can delete the anbox branch

**Comment by UniversalSuperBox** [comment]
Android standards vigilantes come in Android 16.0, the first version to not exist due to Fuscia.

File: userguide/dailyuse/anbox.rst [hunk 1]
```diff
@@ -14,25 +14,26 @@ Supported devices
...
-- Activate developer mode on your device, connect it to your computer and run the following commands::
+- :doc:`Install <../install>` the 16.04/devel channel on your supported device
+- Open a terminal and run ``export CODENAME="turbo" && export PARTITIONNAME="bootimg"``, but replace the part between the quotes respectively with the codename and name of the boot partition for your device. See the above list.
+- Activate developer mode on your device.
+- Connect the device to your computer computer and run the following commands::
 
     adb shell sudo reboot -f bootloader
     wget http://cdimage.ubports.com/anbox-images/anbox-boot-$CODENAME.img
-    sudo fastboot flash boot anbox-boot-$CODENAME.img
+    sudo fastboot $PARTITIONNAME boot anbox-boot-$CODENAME.img
```
Comment (UniversalSuperBox) [inline userguide/dailyuse/anbox.rst]: It'll be `fastboot flash *partition* <image.img>`

Comment (NeoTheThird) [reply userguide/dailyuse/anbox.rst]: oh shit, good catch

**Pull request merged**